src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml

```diff
@@ -16,39 +16,17 @@
              GotKeyboardFocus="Adornment_GotKeyboardFocus"
              Focusable="False"
              UseLayoutRounding="True"
-             x:Name="control"
-             Background="{DynamicResource {x:Static rename:InlineRenameColors.BackgroundBrushKey}}">
+             x:Name="control">
 
     <UserControl.Resources>
         <ResourceDictionary>
-            <ResourceDictionary.MergedDictionaries>
-                <ResourceDictionary Source="../InlineRenameColors.xaml"/>
-            </ResourceDictionary.MergedDictionaries>
-
             <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
-
-            <Style TargetType="CheckBox">
-                <Setter Property="Foreground" Value="{DynamicResource {x:Static rename:InlineRenameColors.CheckBoxTextBrushKey}}"/>
-            </Style>
-
-            <Style TargetType="TextBlock">
-                <Setter Property="Foreground" Value="{DynamicResource {x:Static rename:InlineRenameColors.CheckBoxTextBrushKey}}"/>
-            </Style>
-
-            <Style TargetType="TextBox">
-                <Setter Property="Foreground" Value="{DynamicResource {x:Static rename:InlineRenameColors.TextBoxTextBrushKey}}" />
-                <Setter Property="Background" Value="{DynamicResource {x:Static rename:InlineRenameColors.TextBoxBackgroundBrushKey}}" />
-                <Setter Property="BorderBrush" Value="{DynamicResource {x:Static rename:InlineRenameColors.TextBoxBorderBrushKey}}" />
```

Comment:
Do we still need `InlineRenameColors` if we aren't using it here?

Reply:
Only because I didn't remove usage from the old adornment. I can do a follow up with testing to make sure theme service works there too.